xrspatial/gpu_rtx/hillshade.py

```diff
@@ -114,7 +114,7 @@ def _shade_lambert_kernel(hits, normals, output, H, W, sun_dir, cast_shadows):
         temp = (cos_theta + 1) / 2
 
         if cast_shadows and hits[i, j, 0] >= 0:
-            temp = temp / 2
+            temp = temp / 2.5
```

Comment:
Please add a comment why we're multiplying by this constant here. To me dividing by 2 makes sense but 2.5 is odd

Reply:
Yes, the number is somewhat arbitrary. I think it will be best if I switch it back to being 2.